cypress/integration/dashboard_spec.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-undef */
+/// <reference types="cypress" />
+
+//compound url
+
+//Test Suite
+describe('Home Page Login Form Tests', () => {
+
+    let viewports = [Cypress.env('desktop'),Cypress.env('tablet'),Cypress.env('mobile')];
+
+    beforeEach(() => {
+        cy.visit(Cypress.env('baseUrl'));
+        cy.fixture('user_new.json').as('user_good');
+        cy.fixture('organization_search.json').as('organization');
+    });
+    afterEach(() => {
+        cy.deleteUsersIfExist();
+    });
+
+    viewports.forEach(viewport=>{
+        context(`Testing the ${viewport} Version of the application`,()=>{
+           it('Test Dashboard Elements',()=>{
+            cy.get('@user_good').then(user => {
+                cy.addUser(user).then(()=>{
+                    cy.testDashboardElements(viewport,user);
+                });
+            });
+           });
+           it('Test Dashboard Actions Box1',()=>{
```

Comment:
@Alfredo-Moreira can we get a more descriptive name here please? not sure what we're testing. same for 'box 2'

Reply:
I can address them in the latest PR, but sure. i will give them a more descriptive name. Testing the Org Table display vs the Org Filter functionality